src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.HashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "Cart")
+public class Cart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Czemu masz tutaj mapowanie @OneToOne? Powinno działać bez tego. W sensie Cart powinno mieć po prostu pole Long id i tyle bez relacji.

Reply:
Zastosowałem tutaj mapowanie @OneToOne, ponieważ zamówienie ma być tworzone na podstawie koszyka. Ale jeżeli nie jest to potrzebne to oczywiście usunę